Remotion/ObjectBinding/Web/Res/HTML/BocAutoCompleteReferenceValue.jquery.js

```diff
@@ -1044,7 +1045,7 @@
                 activeItem.addClass (CLASSES.ACTIVE);
                 activeItem.attr("aria-selected", "true");
 
-                select();
+                select(false);

```

Comment:
That's nice. (see comment in next line). Please create a ticket for this and we schedule it for after when this PR is done. We might be able to get rid of line 1050 and pass {{true}} instead. Will require detailed analysis, of course.

Reply:
Added https://re-motion.atlassian.net/browse/RM-7546.